client/app/components/Chart/index.jsx

```diff
@@ -16,9 +16,13 @@ type chartShape = {
 const colorSchemes = ['#6D0839', '#66118', '#7F503F', '#775577', '#CCAADD'];
 
 export function Chart({ chartType, ...props }: chartShape) {
-  return chartType === 'Line' ? (
-    <LineChart {...props} colors={colorSchemes} />
-  ) : (
-    <AreaChart {...props} colors={colorSchemes} />
+  return (
+    <div role="img" aria-label="Canvas chart">
```

Comment:
I would actually switch this to `role="presentation"`and exclude an aria-label. In the future, we should provide a caption for screen-reader users to be able to describe the chart.